packages/client/src/utils/index.ts

```diff
@@ -95,6 +95,20 @@ export const isValidPostalcode = (value?: string) => {
   return !!(value && value.toString().trim().match(postalCodeRegex));
 };
 
+/**
+ *
+ * This function transforms some cases where a user for example makes a typo or makes a strange space combination to a valid option.<
+ *
+ * @param {string} value
+ */
+export const transformHousenumberToValid = (value: string) =>
+  !/[a-z]/i.test(value) // First we test if your string contains a letter. If not, return as is, there is no need processing it
+    ? value
+    : (value?.toUpperCase().match(/[^a-z\s]+|[a-z]/gi) || []) // tokenize the string into letter/non-letter & non-whitespace chunks
```

Comment:
`/[^a-z\s]+|[a-z]/gi` is the same as `/[^A-Z\s]+|[A-Z]/g)` since you're already transformed it `toUpperCase()`.